include/linux/vfs_compat.h

```diff
@@ -74,10 +74,16 @@ truncate_setsize(struct inode *ip, loff_t new)
  * 4.12 - x.y, super_setup_bdi_name() new interface.
  */
 #if defined(HAVE_SUPER_SETUP_BDI_NAME)
+extern atomic_long_t zfs_bdi_seq;
+
 static inline int
 zpl_bdi_setup(struct super_block *sb, char *name)
 {
-	return (super_setup_bdi_name(sb, name));
+	char tmp[32];
+
+	sprintf(tmp, "%.28s%s", name, "-%d");
+	return super_setup_bdi_name(sb, tmp,
+	    atomic_long_inc_return(&zfs_bdi_seq));
```

Comment:
@tuxoko did you mean something like the following?

```c
super_setup_bdi_name(sb, "%.28s-%ld", name, atomic_long_inc_return(&zfs_bdi_seq));
```

I basically just copy/pasted the code from https://github.com/zfsonlinux/zfs/blob/5547c2f1bf49802835fd6c52f15115ba344a2a8b/include/linux/vfs_compat.h#L85, should we update that too?